Sources/StreamChatUI/Common Views/CurrentChatUserAvatarView_Tests.swift

```diff
@@ -0,0 +1,86 @@
+    var customAvatarView: CustomCurrentChatUserAvatarView! = CustomCurrentChatUserAvatarView().withoutAutoresizingMaskConstraints
+    var defaultAppearanceHookAvatarView: CurrentChatUserAvatarView! = CurrentChatUserAvatarView().withoutAutoresizingMaskConstraints
+
+    var mockedCurrentUserController: CurrentChatUserController_Mock<NoExtraData>! = .mock()
+
+    override func setUp() {
+        super.setUp()
+        UIConfig.default.currentUser.currentUserViewAvatarView.defaultAppearance.rules.removeAll()
+    }
+
+    func test_customApperanceHook_currentChatUserAvatarView_withImage_looksAsExpected() {
+        UIConfig.default.currentUser.currentUserViewAvatarView.defaultAppearance.addRule {
+            $0.avatarView.imageView.isOpaque = true
+            $0.backgroundColor = .brown
+            $0.avatarView.imageView.backgroundColor = .cyan
+        }
+
+        mockedCurrentUserController.currentUser_mock = .init(
+            id: "Yoda",
+            imageURL: TestImages.yoda.url
+        )
+        defaultAppearanceHookAvatarView.controller = mockedCurrentUserController
+
+        AssertSnapshot(defaultAppearanceHookAvatarView)
+    }
+
+    func test_customApperanceHook_currentChatUserAvatarView_withoutImage_looksAsExpected() {
+        UIConfig.default.currentUser.currentUserViewAvatarView.defaultAppearance.addRule {
+            $0.avatarView.imageView.isOpaque = true
+            $0.backgroundColor = .brown
+            $0.avatarView.imageView.backgroundColor = .cyan
+        }
+
+        AssertSnapshot(defaultAppearanceHookAvatarView)
+    }
+
+    func test_defaultAppearance_currentChatUserAvatarView_withImage_looksAsExpected() {
+        mockedCurrentUserController.currentUser_mock = .init(
+            id: "Yoda",
+            imageURL: TestImages.yoda.url
+        )
+        defaultAvatarView.controller = mockedCurrentUserController
+        AssertSnapshot(defaultAvatarView)
+    }
+
+    func test_defaultAppearance_currentChatUserAvatarView_withoutImage_looksAsExpected() {
+        AssertSnapshot(defaultAvatarView)
+    }
+
+    func test_setupAppearance_currentChatUserAvatarView_withImage_looksAsExpected() {
+        mockedCurrentUserController.currentUser_mock = .init(
+            id: "Yoda",
+            imageURL: TestImages.yoda.url
+        )
+        customAvatarView.controller = mockedCurrentUserController
+        AssertSnapshot(customAvatarView)
+    }
+
+    func test_setupAppearance_currentChatUserAvatarView_withoutImage_looksAsExpected() {
+        AssertSnapshot(customAvatarView)
+    }
+}
+
+class CustomCurrentChatUserAvatarView: CurrentChatUserAvatarView {
+    override func setUpAppearance() {
+        super.setUpAppearance()
+
+        avatarView.imageView.isOpaque = true
+        backgroundColor = .green
+        avatarView.imageView.backgroundColor = .blue
+        avatarView.imageView.alpha = 0.2
+    }
+}
```

Comment:
Will be changed to `setupLayout`.

Reply:
Changed to `setUpLayout` and `setUpAppearance`

Reply:
It actually makes sense to change to `setupLayout` ?  setup is 1 word not 2 🤔